Modules/selectmodule.c

```diff
@@ -392,33 +447,31 @@ ushort_converter(PyObject *obj, void *ptr)
     return 1;
 }
 
-PyDoc_STRVAR(poll_register_doc,
-"register(fd [, eventmask] ) -> None\n\n\
-Register a file descriptor with the polling object.\n\
-fd -- either an integer, or an object with a fileno() method returning an\n\
-      int.\n\
-events -- an optional bitmask describing the type of events to check for");
+/*[clinic input]
+select.poll.register
+
+    fd: fildes
+      either an integer, or an object with a fileno() method returning an int
+    eventmask: object(converter="ushort_converter", type="unsigned short", c_default="POLLIN | POLLOUT | POLLPRI") = POLLIN | POLLOUT | POLLPRI
```

Comment:
Good opportunity to register the ushort converter.

Reply:
Please keep flags in this order: POLLIN | POLLPRI | POLLOUT. It seems like POLLPRI is more related to POLLIN, than POLLOUT. See https://bugs.python.org/issue30844 :-)